server_code/accounts.py

```diff
@@ -1,185 +1,237 @@
-import anvil.users
+import anvil.google.auth
 import anvil.server
 import anvil.tables
-from anvil.tables import app_tables, order_by
-import anvil.google.auth
 import anvil.tables.query as q
+import anvil.users
 from anvil import secrets
+from anvil.tables import app_tables, order_by
+from anvil_extras.logging import TimerLogger
+
+from . import helper as h
 from . import parameters as p
 from . import portable as port
-from .exceptions import InvalidInviteError
 from . import server_misc as sm
-from . import helper as h
+from .exceptions import InvalidInviteError
 from .server_misc import authenticated_callable
-from anvil_extras.server_utils import timed
-from anvil_extras.logging import TimerLogger
 
 
 @anvil.server.callable
 def get_user(allow_remembered=True, with_id=False):
-  if with_id:
-    user = anvil.users.get_user(allow_remembered)
-    return user, user.get_id() if user else ""
-  else:
-    return anvil.users.get_user(allow_remembered)
+    if with_id:
+        user = anvil.users.get_user(allow_remembered)
+        return user, user.get_id() if user else ""
+    else:
+        return anvil.users.get_user(allow_remembered)
+
 
```

Comment:
Let's not change the indentation in this PR. That should be in a separate PR (could be together with sorting imports).